test/source/tests/decrypt.ts

```diff
@@ -47,6 +48,17 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithBrowser: Te
       await inboxPage.close();
     }));
 
+    ava.default('decrypt - encrypted text inside "message" attachment is correctly decrypted', testWithBrowser('ci.tests.gmail', async (t, browser) => {
+      const acctEmail = 'ci.tests.gmail@flowcrypt.test';
+      const key = Config.key('flowcrypt.compatibility.1pp1')!;
+      await SettingsPageRecipe.addKeyTest(t, browser, acctEmail, key.armored!, key.passphrase, {}, false);
+      await InboxPageRecipe.checkDecryptMsg(t, browser, {
+        acctEmail,
+        threadId: '184a474fc1bd59b8',
+        expectedContent: 'This message contained the actual encrypted text inside a "message" attachment.'
+      });
```

Comment:
This test would succeed on `master`, this is how this message is rendered on `master` (and the live gmail test).
![image](https://user-images.githubusercontent.com/23335944/203713491-0fa1c612-9afd-4da1-b0f7-9574cfa02bf7.png)
But on this branch the `message` is considered to be non-encrypted
![image](https://user-images.githubusercontent.com/23335944/203713722-d743855e-918a-48d4-83a6-47ffd0fd4991.png)
So the gmail rendering and inbox-rendering have diverged, that's not too good.

Reply:
On the opposite, we want the non-encrypted rendering of the `message` in case there is any text in the message body. So we should add such a message to the tests too and make sure it's rendered like this:
![image](https://user-images.githubusercontent.com/23335944/203714650-399a6911-d755-470c-8d91-97b10805c190.png)

Reply:
You are right. Let me fix it